contrib/devtools/test_utxo_snapshots.sh

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env bash
+# Demonstrate the creation and usage of UTXO snapshots.
+#
+# A server node starts up, IBDs up to a certain height, then generates a UTXO
+# snapshot at that point.
+#
+# The server then downloads more blocks (to create a diff from the snapshot).
+#
+# We bring a client up, load the UTXO snapshot, and we show the client sync to
+# the "network tip" and then start a background validation of the snapshot it
+# loaded. We see the background validation chainstate removed after validation
+# completes.
+#
+
+export LC_ALL=C
+set -e
+
+BASE_HEIGHT=${1:-30000}
+INCREMENTAL_HEIGHT=20000
+FINAL_HEIGHT=$(("$BASE_HEIGHT" + "$INCREMENTAL_HEIGHT"))
```

Comment:
Getting `syntax error: operand expected (error token is ""30000" + "20000"")` on macOS
```suggestion
FINAL_HEIGHT=$(($BASE_HEIGHT + $INCREMENTAL_HEIGHT))
```

Reply:
Which version of Python are you using?

Reply:
@jamesob line 1 (`#!/usr/bin/env bash`) sets the interpreter to Bash regardless of what the system default is.

@MarcoFalke you were right! macOS comes with an outdated version of bash (`3.2.57`). I have updated to `5.2.15`. Script runs like a charm now. 

Thanks for the help!

Reply:
Fixed, thanks all.